queries.sql

```diff
@@ -56,4 +56,13 @@ select min(weight_kg) from animals where species = 'Pokemon';
 select min(weight_kg) from animals where species = 'Digimon';
 
 select avg(escape_attempts) from animals where species = 'Pokemon' and date_of_birth > '1989-12-31' and date_of_birth < '2001-01-01';
-select avg(escape_attempts) from animals where species = 'Digimon' and date_of_birth > '1989-12-31' and date_of_birth < '2001-01-01';
\ No newline at end of file
+select avg(escape_attempts) from animals where species = 'Digimon' and date_of_birth > '1989-12-31' and date_of_birth < '2001-01-01';
+
+select animals.name from animals join owners on owners.id = animals.owners_id where owners.full_name = 'Melody Pond';
+select animals.name from animals join species on species.id = animals.species_id where species.name = 'Pokemon';
+select owners.full_name as owners, animals.name from animals full outer join owners on owners.id = animals.owners_id;
+select species.name as species, count(animals.name) from animals join species on species.id = animals.species_id group by species.name;
+select owners.full_name as owner, animals.name as name from animals join species on species.id = animals.species_id
+join owners on owners.id = animals.owners_id where owners.full_name = 'Jennifer Orwell' and species.id = 2;
+select animals.name from animals join owners on owners.id = animals.owners_id where animals.escape_attempts = 0 and owners.full_name = 'Dean Winchester';
+select owners.full_name as owners, count(animals.name) as count from animals join owners on owners.id = animals.owners_id group by owners.full_name;
```

Comment:
- [ ] Please kindly make this query return only one owner as the answer to who owns the most animals should not be more than one. You can arrange the output here in descending order and take only the first output using limit.

Reply:
Thanks for the suggestion, the requested changes have been made, and thanks for your advice on the limit!

![capture13](https://user-images.githubusercontent.com/90111138/222551323-cc12321c-8ceb-4cbe-be22-2e9e125b8f3d.PNG)